foodalert/static/foodalert/js/components/food-categories-template.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+    <b-container class="px-4 pt-3">
+    <b-row class="justify-content-center">
+        <b-col md="8">
+            <p>Select all that apply to your food.</p>
+            <b-form-group>
+              <b-form-checkbox
+                v-model="selected"
+                key="non-perishable"
+                value="non-perishable"
+                :disabled="selected.includes('none') || selected.includes('home')"
+              >{{"My food is non-perishable. "}}<a href="#" @click.prevent.stop="tooltip1 = !tooltip1">Examples</a></b-form-checkbox>
+              <div class="tooltip-example" v-if="tooltip1">Candy, beverages (pasteruized, canned, or bottled), chips, dips.</div>
+              <b-form-checkbox
+                v-model="selected"
+                key="pre-packaged"
+                value="pre-packaged"
+                :disabled="selected.includes('none') || selected.includes('home')"
+              >{{"My food was commercially pre-packaged. "}}<a href="#" @click.prevent.stop="tooltip2 = !tooltip2">Examples</a></b-form-checkbox>
+              <div class="tooltip-example" v-if="tooltip2"> Wrapped or boxed baked goods (cakes, pies), chips, store-bought ice cream.</div>
+              <b-form-checkbox
+                v-model="selected"
+                key="home"
+                value="home"
+                :disabled="selected.includes('non-perishable') || selected.includes('pre-packaged') || selected.includes('none')"
+              >{{"My food was prepared at home."}}</b-form-checkbox>
+              <b-form-checkbox
+                v-model="selected"
+                key="none"
+                value="none"
+                :disabled="selected.includes('non-perishable') || selected.includes('pre-packaged') || selected.includes('home')"
+              >{{"None of the above."}}</b-form-checkbox>
+            </b-form-group>
+
+            <div class="mt-5">
+               <b-row align-h="between">
+                 <b-col md="4" lg="3" order-md="2">
+                   <b-button class="mb-3" type="submit" block variant="primary" @click="getNextPage()">Continue</b-button>
+                 </b-col>
+                 <b-col md="4" lg="3" order-md="1">
+                   <b-button class="hh-back-button" type="submit" block variant="light" @click="getBackPage()">Back</b-button>
+                 </b-col>
+               </b-row>
+             </div>
+        </b-col>
+    </b-row>
+    </b-container>
+</template>
+
+<script type="text/javascript">
+  export default {
+    methods: {
+        getNextPage() {
+            if (this.selected.includes('none') || this.selected.includes('home')) {
+                this.$router.push({ name: 'notfound' });
```

Comment:
Currently this is set to navigate to the "notfound" page as the "exit app" page has not yet been build out. Whoever builds out that page should make sure this route will switch over to the route for the new page.